streaming/base/dataset.py

```diff
@@ -98,30 +98,29 @@ class StreamingDataset(IterableDataset):
         }
 
     StreamingDataset init takes three kinds of arguments:
-      * One or more Streams (you must provide either ``streams`` or ``remote``/``local``):
-          * ``streams``
-          * ``remote``
-          * ``local``
-      * Knobs to control streaming behavior, which, if multiple Streams are provided, become defaults
-        applied to them:
-          * ``split``
-          * ``download_retry``
-          * ``download_timeout``
-          * ``validate_hash``
-          * ``keep_zip``
-          * ``keep_raw``
-      * How to iterate (controlling prefetching, partitioning, and shuffling):
-          * Prefetching:
-              * ``predownload``
-          * Partitioning:
-              * ``partition_algo``
-              * ``num_canonical_nodes``
-              * ``batch_size``
-          * Shuffling:
-              * ``shuffle``
-              * ``shuffle_algo``
-              * ``shuffle_seed``
-              * ``shuffle_block_size``
+        * One or more Streams (you must provide either ``streams`` or ``remote``/``local``):
```

Comment:
what's the scheme of this indentation change?

Reply:
I think either it has to be 4 spaces or 2 spaces but not mixed. Previously, it was mixed. Here it is 4 space

Reply:
Need to play with this some more, Sphinx was throwing a warning locally so I was messing around with it.

Reply:
was doing 4-space, not counting the preceding "* "

Reply:
Ok think I fixed it, the initial asterisk of the main bullet needs to be aligned with the text body above (no indentation).  Subsequent nested sub bullets need to be separated by a blank line and indented by two spaces.  Gotta love RST